book/graphs/pharmacokinetic/pkpg06.qmd

```diff
@@ -137,7 +139,7 @@ outliers <- function(x) {
   return(x < quantile(x, 0.05) | x > quantile(x, 0.95))
 }
 
-p <- ggplot(anl_x, aes(factor(PARAMCD), ratio, fill = factor(ACTARM), label = USUBJID)) +
+plot <- ggplot(anl_x, aes(factor(PARAMCD), ratio, fill = factor(ACTARM), label = USUBJID)) +
```

Comment:
Snapshot fails because the `.rds` file cannot access `outliers` which is internally defined.
```
! could not find function "outliers"
```

Reply:
Right now the fix is to recreate these locally created objects as a testthat helper file so the tests have access to them.